Sources/BraintreeShopperInsights/BTCreateCustomerSessionApi.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// The API used to create a customer session using the `CreateCustomerSession` GraphQL mutation.
+/// - Warning: This feature is in beta. It's public API may change or be removed in future releases.
+class BTCreateCustomerSessionApi {
+    
+    // MARK: - Properties
+    
+    /// Exposed for testing to get the instance of `BTAPIClient`
+    var apiClient: BTAPIClient
+    
+    /// :nodoc: This typealias is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    typealias CreateCustomerSessionResult = (String?, Error?) -> Void
```

Comment:
Do we need a typealias for this?

Reply:
ahh that's right we don't need it - updated in 4b947e866a98bf0b87067d6f6b4301697bdc1a53